### DISCUSSION TITLE
Gemeinsames Darstellungswerkzeug für geplante tagesaktuelle GDB Abfrage des BEV

### DISCUSSION THREAD

**Discussion by faulgurke** [e-netze/webgis-community#65] [Ideas]
Das BEV plant einen tagesaktuellen Service für GDB Abfragen über den Portalverbund. Einige Länder haben die entsprechende Vereinbarung bereits unterzeichnet.
Wäre es eine Möglichkeit, eine für alle WebGIS Nutzer verwendbare Eigentümerabfrage (über Portalverbundprotokoll) zu realisieren?

**Comment by gis-eni** [comment]
Gibt es ein BEV API für diese Abfrage?

**Comment by gis-eni** [comment]
Tool in die WebGIS Karte integrieren. Eventuell muss man dazu über einen Hotlink eine Javascript Methode aufrufen können.

**Comment by gis-eni** [comment]
Tool befindet sich im Installationsverzeichnis für LR unter `examples`.
Gesamten `e-plugins` Ordner irgendwo hin kopieren (muss über Internet erreichbar sein, Virtuelles Verzeichnis, ...)

Beschreibung unter `e-plugins\webgis\bev_gst\readme.md`
